contributing_to_docs/doc_guidelines.adoc

```diff
@@ -310,6 +391,23 @@ For all of the system blocks including table delimiters, use four characters. Fo
 ---- for code blocks
 ....
 
+Use this format when embedding footnotes into the code block:
```

Comment:
On the Customer Portal builds, having `====` wrapped around it doesn't actually make a difference. On docs.openshift builds, it does show a small line outline around the whole thing, e.g.:

https://docs.openshift.com/container-platform/3.3/admin_guide/manage_authorization_policy.html#managing-role-bindings

Since it only makes a difference in one of the builds, and since it only yields a small cosmetic change at all, I'd suggest for simplicity to just drop using `====` around any codeblocks, callouts or no. And I never liked the previous distinction of our usage being "command syntax" vs "example command/output" either. So yeah, I suggest we just stick with `----` for all codeblocks, no matter the context, and just be done with it.